pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceProducer.java

```diff
@@ -753,11 +760,13 @@ private static void runProducer(int producerId,
             if (null != client) {
                 try {
                     client.close();
-                    PerfClientUtils.exit(1);
                 } catch (PulsarClientException e) {
                     log.error("Failed to close test client", e);
                 }
             }
+            if (!produceEnough) {
```

Comment:
How about move this block to line before `if (null != client)`?  Or catch `Throwable` when client close.

Reply:
why do we move this block to line before if (null != client)?

Reply:
Could `client.close` throw other exceptions , which will cause this block not be executed

Reply:
you are right, i have fixed it.